### DISCUSSION TITLE
Is there a way to combine two different models from two different URDFs together?

### DISCUSSION THREAD

**Discussion by Zionshang** [stack-of-tasks/pinocchio#2314] [Q&A] [answered]
For example, I have two URDFs. One is `quadruped_robot.urdf`, another is `manipulator.urdf`. 
I use `pinocchio::urdf::buildModel()`  to generate two different models.
Is there a way to combine these two models together? The result is a single model.

I notice there is `pinocchio::buildReducedRobot`. Whether there is a method to do the opposite thing?

**Comment by fabinsch** [comment, marked as answer]
Hi @Zionshang,

the method that you are searching for is called `appendModel`, see the docs [here](https://gepettoweb.laas.fr/doc/stack-of-tasks/pinocchio/master/doxygen-html/namespacepinocchio.html#a3593a709375ef09ea794a288dfbfec83).

**Reply by Zionshang** [reply]
THANKS A LOT!!!!!
